tools/commissioner_thci/commissioner_impl.py

```diff
@@ -169,11 +168,12 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
     @classmethod
     def make_local_commissioner(config, simulator):
+        import pexpect
         handler = pexpect.spawn("/bin/bash")
         return OTCommissioner(config, handler, simulator)
 
     @classmethod
-    def make_remote_commissioner(config, serial_handler):
+    def make_harness_commissioner(config, serial_handler):
```

Comment:
the first argument of an `classmethod` should be cls, which will be a reference to the class itself.
you should add the `cls` argument, or use `staticmethod` instead.

Same for `make_local_commissioner`

Reply:
`staticmethod` seems to be what I want.